pkg/webui/lib/shared-messages.js

```diff
@@ -45,7 +45,7 @@ export default defineMessages({
   appEUI: 'AppEUI',
   appEUIJoinEUI: 'AppEUI/JoinEUI',
   appEUIDescription:
-    'The AppEUI is a global application ID that uniquely identifies the owner of the end ­device. AppEUI is used for end devices with LoRaWAN versions 1.0.0-1.0.3, JoinEUI for 1.0.4+.',
+    'The AppEUI uniquely identifies the owner of the end ­device. If no AppEUI is provided by your device manufacturer (usually for development), use 0000000000000000',
```

Comment:
```suggestion
    'The AppEUI uniquely identifies the owner of the end ­device. If no AppEUI is provided by your device manufacturer (usually for development), it can be filled with zeros',
```

I still think this is better than suggesting the full value, which looks odd.

Reply:
true that it looks nicer, but it's a little bit clearer to say "use 0000000000000000," especially because they also have to program that number in to their device.

Reply:
The field description is not a good place to suggest concrete byte values because we always style displayed byte values in a `<SafeInspector />`. My condition to accept displaying the full value is that it is styled in a way that resembles the way that values are displayed in the `<SafeInspector />`.

IMO it's not worth the hassle since the other formulation should be clear enough in conjunction with the fill button.